pre-commit

```diff
@@ -102,6 +102,9 @@ check "Tests" \
 check "Web tests" \
     cargo test -p ipa-core --no-default-features --features "cli web-app real-world-infra test-fixture compact-gate"
 
+check "Web tests (descriptive gate)" \
+    cargo test -p ipa-core --no-default-features --features "cli web-app real-world-infra test-fixture"
+
```

Comment:
maybe a dumb question, but how is descriptive gate related to the active work?

Reply:
It's not, but some of the plumbing I had to change to add the option broke a test case that only runs with descriptive gate, that is run by the coverage job, but not by `pre-commit`, so I added this.